django/core/management/commands/loaddata.py

```diff
@@ -33,6 +33,7 @@ class Command(BaseCommand):
         "No database fixture specified. Please provide the path of at least "
         "one fixture in the command line."
     )
+    private_options = ('commit', 'using')
```

Comment:
What are these options -- I don't see them used -- maybe they are mistakes in the `call_command()`?